monitor/xfeemngr/contract/bound.go

```diff
@@ -128,14 +149,26 @@ func (c BoundFeeOracleV1) BulkSetFeeParams(ctx context.Context, params []binding
 		return errors.Wrap(err, "bulk set fee params")
 	}
 
-	_, err = c.backend.WaitMined(ctx, tx)
+	rec, err := c.backend.WaitMined(ctx, tx)
 	if err != nil {
 		return errors.Wrap(err, "wait mined", "tx", tx.Hash().Hex())
 	}
 
+	spendTotal.WithLabelValues(c.chain.Name, string(c.chain.NativeToken), methodBulkSetFeeParams).Add(totalSpentGwei(tx, rec))
+
 	return nil
 }
 
+// totalSpentGwei returns the total amount spent on a transaction in gwei.
+func totalSpentGwei(tx *ethtypes.Transaction, rec *ethtypes.Receipt) float64 {
+	fees := new(big.Int).Mul(rec.EffectiveGasPrice, new(big.Int).SetUint64(rec.GasUsed))
+	value := new(big.Int).Add(fees, tx.Value())
+	total := new(big.Int).Add(value, fees)
```

Comment:
Not sure about the math here, feels like we are doing: `total := fees + tx.Value + fees`

Reply:
yeah this is mistake